Services/ReportsRepository/classes/class.ilObjReportBase.php

```diff
@@ -214,6 +214,18 @@ public function getVideoLink() {
 		return $this->video_link;
 	}
 
+	public function setPDFLink($pdf_link) {
+		if($pdf_link != "" && !preg_match(self::VIDEO_LINK_REGEX, strtolower($pdf_link))) {
```

Comment:
Leg bitte für diese beiden Stellen (also auch die oben in setVideoLink) mal ein Ticket an. Ich glaube wir könnten diese Funktion schön in eine Lib packen oder eine geeignete Lib einfach einbinden. Dann würden wir auch diese Konstante loswerden, die im moment nicht mehr so clever benannt ist.